lib/rules/boolean-prop-naming.js

```diff
@@ -131,7 +135,7 @@ module.exports = {
         const propName = getPropName(propNode);
         context.report({
           node: propNode,
-          message: `Prop name (${propName}) doesn't match rule (${config.rule})`,
+          message: config.message || `Prop name (${propName}) doesn't match rule (${config.rule})`,
```

Comment:
I think if we're making this configurable, then instead of using a template literal to interpolate `propName` and `config.rule`, we should use eslint's built-in string templating - that way the configurable message can use placeholders for those two things.

Reply:
Adding now.